schematic/manifest/generator.py

```diff
@@ -1489,6 +1490,107 @@ def _handle_output_format_logic(self, output_format: str = None, output_path: st
         # Default return a DataFrame
         else:
             return dataframe
+    
+    @staticmethod
+    def create_single_manifest(jsonld: str, data_type: str, access_token:str=None, dataset_id:str=None, strict:bool=True, title:str=None, output_format:str=None, use_annotations:bool=False) -> Union[str, pd.DataFrame, BinaryIO]:
+        """Create a single manifest
```

Comment:
Could you also add to the comment on what parameters lead to the different output types? `str, pd.DataFrame, BinaryIO`

Reply:
Could you also change `output_format:str=None` to `output_format:Literal[None, "google sheet", "excel", "dataframe"]=None`. This will have mypy enforce that anythign calls this method will use one of those values.

Reply:
What happens when `output_format= None`?

Reply:
Thanks @andrewelamb for the comments. When `output_format` is None, then the function would return a google sheet link.

Reply:
For the bool types I see you changed them to `Optional[bool]=True`. I would only do this if you treated `None` inputs differently than `False` inputs. ie you have somethign like 

`if Optional is None:`

If not I would change it back to `bool=True`

Reply:
Thanks for the comment @andrewelamb. There is code to deal with situations where title = None, so I didn't make any changes of typing of `title`. I also updated `output_format` parameter to not take in None as an input and set the default as "google_sheet".